### DISCUSSION TITLE
Question about basic auth

### DISCUSSION THREAD

**Discussion by pimpmyname2** [ZoeyVid/NPMplus#565] [Q&A] [answered]
I was wondering if its possible to use basic authentication for a proxied site like for an example for https://example.com/wp-login.php

**Comment by Zoey2936** [comment]
using advanced config yes, without, no

**Reply by pimpmyname2** [reply]
How?

**Reply by Zoey2936** [reply, marked as answer]
create a custom location like this:

![grafik](https://github.com/ZoeyVid/NPMplus/assets/75573284/78342916-31e4-4ac5-b352-43a8f3db4039)

**Reply by pimpmyname2** [reply]
Thanks
